pipelines/HSC/DRP-Prod.yaml

```diff
@@ -1,8 +1,29 @@
 description: The DRP pipeline specialized for large data release productions
 instrument: lsst.obs.subaru.HyperSuprimeCam
 imports:
-  location: "$DRP_PIPE_DIR/pipelines/HSC/DRP-RC2.yaml"
+  - location: "$DRP_PIPE_DIR/ingredients/HSC/DRP.yaml"
+    exclude: faro_all
+tasks:
+  fgcmFitCycle:
+    class: lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask
+    config:
+      doMultipleCycles: true
```

Comment:
Will do, especially since 5 was just what you wanted for the test run, and that's now done.  I'm assuming that change goes hand-in-hand with `fgcmOutputProducts.cycleNumber=6`.

Reply:
Yes!  Good catch.